app/views/publisher/_publisher.html.haml

```diff
@@ -51,30 +51,23 @@
         .loader
           .spinner
       .options_and_submit.col-sm-12
-        .public_toggle
+        .public_toggle.clearfix
           .btn.btn-default.pull-left#hide_publisher{title: t('shared.publisher.discard_post')}
             %span.text
               =t('cancel')
 
           .btn-toolbar.pull-right
-            %span#publisher_service_icons.btn-group
-              - if current_user.services
-                - for service in current_user.services
-                  = service_button(service)
-              .btn.btn-link{ class: "question_mark",
-                             title: t("shared.public_explain.manage"),
-                             data:  {toggle: "modal", target: "#publicExplainModal"}}
-                %i.entypo-cog
-
             = render partial: "publisher/aspect_dropdown", locals: { :selected_aspects => selected_aspects }
+            %button.btn.btn-default.btn-group.post_preview_button= t('shared.publisher.preview')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.